addons/mass_mailing/tests/test_mailing_saving.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo.tests
+
+
+@odoo.tests.tagged('-at_install', 'post_install')
+class TestUi(odoo.tests.HttpCase):
+    def mass_mailing_saving_tour(self):
+        self.start_tour("/web", 'mass_mailing_saving_tour', login="admin")
+        mail = self.env['mail.mail'].search([('subject', '=', 'Test')])[0]
+        self.assertNotEqual(mail.body_arch, mail.body_html)
```

Comment:
Not sure what you are trying to check here: no comment, no message if crash , ... nobody would understand why this test crashes.